src/biotite/structure/io/pdbx/convert.py

```diff
@@ -131,12 +131,18 @@ def get_sequence(pdbx_file, data_block=None):
         file.
         If the data block object is passed directly to `pdbx_file`,
         this parameter is ignored.
+    chain_ids : bool, optional
+        If True, a dictionary mapping chain IDs to sequences is returned.
+        If False or not given, all sequences are returned.
+        Default is False.
 
     Returns
     -------
-    sequences : list of Sequence
-        The protein and nucleotide sequences for each entity
-        (equivalent to chains in most cases).
+    sequences : list of Sequence or dict
+        If `chain_ids` is False, returns a list of protein and nucleotide 
+        sequences for each entity.
+        If `chain_ids` is True, returns a dictionary where each key is a 
+        chain ID and each value is the corresponding sequence.
```

Comment:
I think we should note here that the chain IDs correspond to `atom_site.auth_asym_id`.